cores/esp32/HardwareSerial.cpp

```diff
@@ -127,11 +124,8 @@ HardwareSerial Serial2(2);
 
 void serialEventRun(void)
 {
-#if ARDUINO_USB_CDC_ON_BOOT //Serial used for USB CDC
-    if(Serial0.available()) serialEvent();
-#else
+    // Serial is always defined as something (UART0 or USBSerial)
     if(Serial.available()) serialEvent();
```

Comment:
Should be `Serial0.available()`

Reply:
I was thinking about it yesterday. If I use `Serial.available()`, it works with the CDC as well... 
`Serial0` will limit it to UART only. 
Any opinion?